qt/widgets/sliceviewer/src/LineViewer.cpp

```diff
@@ -246,15 +246,15 @@ void LineViewer::updateFreeDimensions() {
 //-----------------------------------------------------------------------------------------------
 /** Show the start/end/width points in the GUI */
 void LineViewer::updateStartEnd() {
-  for (int d = 0; d < int(m_ws->getNumDims()); d++) {
-    m_startText[d]->setText(QString::number(m_start[d]));
-    m_endText[d]->setText(QString::number(m_end[d]));
-    m_thicknessText[d]->setText(QString::number(m_thickness[d]));
+  const size_t ndims{m_ws->getNumDims()};
+  for (size_t d = 0; d < ndims; d++) {
+    if (d < m_start.getNumDims())
+      m_startText[d]->setText(QString::number(m_start[d]));
+    if (d < m_end.getNumDims())
+      m_endText[d]->setText(QString::number(m_end[d]));
+    if (d < m_thickness.getNumDims())
+      m_thicknessText[d]->setText(QString::number(m_thickness[d]));
   }
-  ui.textPlaneWidth->setText(QString::number(m_planeWidth * 2.));
-
-  // Now show the width
-  this->updateBinWidth();
```

Comment:
I cannot see why this disappeared, can you elaborate?

Reply:
You're right this should not have disappeared. I pushed some changes over the weekend that I hadn't tested yet. I will reinstate them.